### ISSUE TITLE
urine lipocalin 2 terms

### ISSUE THREAD

**Issue by slaulederkind** [obophenotype/mammalian-phenotype-ontology#3271]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1. abnormal urine lipocalin 2 level
2. decreased urine lipocalin 2 level
3. increased urine lipocalin 2 level

>2. Synonyms (e.g., absent spleen)

1. abnormal urine Lcn2 level; abnormal urine NGAL level; abnormal urine neutrophil gelatinase-associated lipocalin level
2. reduced urine Lcn2 level; reduced urine lipocalin 2 level; decreased urine NGAL level; decreased urine neutrophil gelatinase-associated lipocalin level
3. increased urine Lcn2 level; increased urine NGAL level; increased urine neutrophil gelatinase-associated lipocalin level

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1. deviation from the normal urinary levels of lipocalin 2, a protein that plays a role in innate immunity by limiting bacterial growth as a result of sequestering iron-containing siderophores
2. reduction in the amount in the urine of lipocalin 2, a protein that plays a role in innate immunity by limiting bacterial growth as a result of sequestering iron-containing siderophores
3. increase in the amount in the urine of lipocalin 2, a protein that plays a role in innate immunity by limiting bacterial growth as a result of sequestering iron-containing siderophores

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

PMID:29437896

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1. abnormal urine protein level (MP:0006315)
2. abnormal urine lipocalin 2 level (new)
3. abnormal urine lipocalin 2 level (new)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0002-9553-7227

**Comment by sbello** [comment]
MP:0020987 - abnormal
MP:0020988 - decreased
MP:0020989 - increased
